src/components/FloatingActionButton.js

```diff
@@ -63,22 +73,25 @@ class FAB extends PureComponent {
         });
 
         return (
-            <Tooltip absolute text={this.props.translate('common.new')}>
-                <AnimatedPressable
-                    accessibilityLabel={this.props.accessibilityLabel}
-                    accessibilityRole={this.props.accessibilityRole}
-                    onPress={this.props.onPress}
-                    style={[
-                        styles.floatingActionButton,
-                        StyleUtils.getAnimatedFABStyle(rotate, backgroundColor),
-                    ]}
-                >
-                    <AnimatedIcon src={Expensicons.Plus} fill={fill} />
-                </AnimatedPressable>
-            </Tooltip>
+            <KeyboardAvoidingView behavior="padding">
```

Comment:
This was a mistake to add because it created duplicate keyboard avoiding views. This was caught in the issue here: https://github.com/Expensify/App/issues/10905

Reply:
The other one is in `BaseScreenWrapper`